dist/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+<script defer src="main.js"></script></head>
+<body>
+    <header>
+        <h1>Choose and Ingredient</h1>
+        <nav>
+            <div id="logo"></div>
+            <div class="menu">
+                <h2><a href="#" class="ingredient active">Pork</a></h2>
+            </div>
+        </nav>
+    </header>
```

Comment:
In order to have a nice layout and design, it is better to have a well-structured HTML file.

- [ ] Please consider redesigning the header in order to contain in the top left the logo and on the centre or towards the right side the navigation menu (with dummy links it will do). The`h1` tag will be better to be part of the main section on the web page and not of the header.